mmv1/products/compute/InstanceTemplate.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name: 'InstanceTemplate'
```

Comment:
looks good but can you address the missing fields from the VCR tests?

https://github.com/GoogleCloudPlatform/magic-modules/pull/11932#issuecomment-2413701544

Reply:
not really sure because the tests are autogenerated from the yaml file. There aren't really any docs on doing custom IAM tests like that in the magic-modules documentation, so don't know if it's possible. 

If you know a solution to do custom test cases for iam resources then i'd code that in.